pkg/server/backuprestoreserver.go

```diff
@@ -224,24 +210,28 @@ func (b *BackupRestoreServer) runServer(ctx context.Context, restoreOpts *brtype
 			}
 			_ = miscellaneous.SleepWithContext(ctx, retryTimeout)
 		}
-	} else {
-		// when OriginalClusterSize = 1
-		m := member.NewMemberControl(b.config.EtcdConnectionConfig)
-		err := retry.OnError(retry.DefaultBackoff, func(err error) bool {
-			return err != nil
-		}, func() error {
-			cli, err := etcdutil.NewFactory(*b.config.EtcdConnectionConfig).NewCluster()
-			if err != nil {
-				return err
-			}
-			_, err = m.UpdateMemberPeerURL(ctx, cli)
+	}
```

Comment:
We don't need this block of promoting member. We are already promoting member downwards. Can we remove it?

Reply:
Yes this was removed in this PR. Maybe i did not understand your suggestion. If you view the file you will see that this block was removed. Now we always want to promote a member irrespective of cluster size.

Reply:
I think what @abdasgupta is saying that we are promoting the learner in leader-election as well. check [here](https://github.com/gardener/etcd-backup-restore/blob/master/pkg/leaderelection/leaderelection.go#L153-L157).
@aaronfern can we remove this from here or may be @abdasgupta you can address this in your PR ?

Reply:
Better if you remove PromoteMember from here. But keep the UpdateMemberPeerURL